client/dfclient/src/state_manager.cpp

```diff
@@ -41,15 +41,26 @@ StateManager::StateManager() {
 		}
 		file = soundDir.readNext();
 	}
+
+	_resources._sounds.erase("you-failed.wav"); // temporary disable, it will be useful as a gameover sound effect
```

Comment:
maybe just remove it from the data repository, I don't see much use for this sound effect (and also it's copyrighted af)

Reply:
I thought of it as a thing that could play when you finish last. Copyright is a point, but... nearly every single sound we use in this game is copyrighted xDD All I'm counting on is the meme-ish-ness of the game when it comes to sound. But we can remove this if you want

Reply:
Copyright point taken, all of the sounds are copyrighted, so this doesn't matter. However I don't want to be punishing players that were last, it would probably be better to award the ones that were first instead. Please remove